Resources/Prototypes/DeltaV/Roles/Jobs/Justice/chief_justice.yml

```diff
@@ -0,0 +1,58 @@
+- type: job
+  id: ChiefJustice
+  name: job-name-chiefjustice
+  description: job-description-chiefjustice
+  playTimeTracker: JobChiefJustice
+  antagAdvantage: 6 # DeltaV - Reduced TC: Head of Staff
```

Comment:
Ignoring that they can't roll antag, this is insane.

Reply:
I think that may be a remnant from when Phil was copying over the yaml from roles. Will remove it